### ISSUE TITLE
DEFINE: DEEP FACTORIZED INPUT TOKEN EMBEDDINGS FOR NEURAL SEQUENCE MODELING

### ISSUE THREAD

**Issue by AkiraTOSEI** [AkiraTOSEI/ML_papers#39]
# TL;DR

Proposes DeFINE to refine language embedding vectors. It groups and integrates the channels of tokens in a hierarchical manner. The number of parameters can be reduced, and the effect is confirmed by using both LSTM and Transformer in a sequence-type model
![image](https://user-images.githubusercontent.com/24524018/89788333-f360c880-db59-11ea-9138-4d6758935d2f.png)
![image](https://user-images.githubusercontent.com/24524018/89788344-f6f44f80-db59-11ea-9ed9-16e06803ebeb.png)

# Why it matters:

# Paper URL
https://arxiv.org/abs/1911.12385

# Submission Dates(yyyy/mm/dd)
2020/11/27

# Authors and institutions
Sachin Mehta, Rik Koncel-Kedziorski, Mohammad Rastegari, Hannaneh Hajishirzi
- University of Washington
- Allen Institute for AI

# Methods

# Results
![image](https://user-images.githubusercontent.com/24524018/89788382-08d5f280-db5a-11ea-9ead-61959cdb0231.png)

# Comments
ICLR2020
